docs/configuration/theme.toml.md

```diff
@@ -1,11 +1,14 @@
 # theme.toml
 
-This file is used to theme joshuto
+This file is used to theme Joshuto.
 
-## Style
+As of now, all theming items are “text style” items where one can define the
```

Comment:
Nit, I would use "" instead of “”. Unless you have a reason to use those quotes.